.make.defaults

```diff
@@ -305,7 +306,10 @@ endif
 	if [ ! -z "$(EXTRA_INDEX_URL)" ]; then				\
 		extra_url='--extra-index-url $(EXTRA_INDEX_URL)';	\
 	fi;								\
-	pip install $(PIP_INSTALL_EXTRA_ARGS) $${extra_url}  -e $(PYTHON_PROJECT_DIR);
+	if [ -e $(PYTHON_PROJECT_DIR)/requirements.txt ]; then \
```

Comment:
how sure are you that this is necessary now that we also copy the requirements.txt from the python dir into python-transform installation dir.  I'm not sure it is needed anymore.

Reply:
I will be revisiting this and other makefile stuff in a new PR. For now, since it does not break anything, I will keep it.